templates/config.fish

```diff
@@ -6,8 +6,8 @@ set -g OMF_CONFIG {{OMF_CONFIG}}
 
 ### Configuration required to load oh-my-fish ###
 # Note: Only add configurations that are required to be set before oh-my-fish is loaded.
```

Comment:
Most users don't need to add this file and it's going to be well explained on the README when necessary.